codecov.yml

```diff
@@ -1,2 +1,23 @@
+# Documentation: https://docs.codecov.io/docs/codecov-yaml
 ignore:
   - "api/v1alpha1/zz_generated.deepcopy.go" # generated file
+
+# general codecov settings
+codecov:
+    notify:
+        require_ci_to_pass: false
```

Comment:
what does this mean?

Reply:
_Should Codecov wait for all other statuses to pass before sending its status._

It is only about notifications.

Reply:
So the statuses are what, patch and project? This just means don't wait for the project result to add the patch result (and vice versa)?

Reply:
that's right.

Reply:
According to the docs, `require_ci_to_pass` is used to `Show the Codecov [status without waiting](https://docs.codecov.com/docs/codecovyml-reference#codecovrequire_ci_to_pass) for other status checks to complete or pass.
```
So I'm not getting why it works. We currently have a CI step that never completes or passes but we still get the notification. What am I missing?

Reply:
I understood it only regards to notifications.

Reply:
thinking now this line is useless. I will remove it.